bundle/edu.gemini.catalog/src/main/scala/edu/gemini/catalog/votable/VoTableClient.scala

```diff
@@ -131,7 +131,6 @@ trait CachedBackend extends VoTableBackend {
 
   // Cache the query not the future so that failed queries are executed again
   override protected [votable] def doQuery(query: CatalogQuery, url: URL)(ec: ExecutionContext): Future[QueryResult] = Future {
-    Logger.getLogger(getClass.getName).info(s"Starting BAGS lookup on ${Thread.currentThread}")
```

Comment:
Perhaps we should fix the message and maybe lower the log level?  I don't know but it seems vaguely useful to have the option of logging this particularly given the issue with AGS estimations.